bran/helpers/blender.py

```diff
@@ -0,0 +1,102 @@
+from bran.helpers.utils import get_security_groups, get_local_awsconfig, list_to_choices
+import os
+
+def get_blender_questions():
+    """
+    Constructs and returns questions for cli to set up ec2 instance properties for image generation with Blender.
+
+    Return:
+        questions ([dicts]): a list of dictionaries with each dictionary representing
+            a different question
+    """
+
+    amis = ['Blender:ami-0c66a2d9734c3f1aa', "Blender-GPU:ami-0e96d01a5638bccd8"]
```

Comment:
It may be useful to allow a custom ami as well?

Reply:
Good idea. I'll look into this